### ISSUE TITLE
[Bug]: Editor Color Scheme being reset every time IntelliJ is opened

### ISSUE THREAD

**Issue by spencerhakim** [mallowigi/material-theme-issues#180]
### Describe the bug

Like the title says, every time I open IntelliJ the Editor Color Scheme is being reset to one provided by Material Theme UI instead of the one I previously had selected, which is not provided by Material Theme UI. Encountered this issue after updating to the latest plugin, and it was presumably introduced by either Color Scheme-related changes made in 7.5.2 or the fix for #176.

### To Reproduce

1. Set Editor Color Scheme to a non-Material Theme UI scheme
2. Close/re-open IntelliJ
3. Selected Editor Color Scheme has been changed

### Expected behavior

1. Plugin should not change Editor Color Scheme unprompted

### Material Theme Version

v7.5.2.1

### IDE

IntelliJ IDEA

**Comment by serpro69** [comment]
I have the same issue after upgrading to latest version

**Comment by mallowigi** [comment]
Thanks for reporting. I haven't thought of that scenario.

**Comment by mallowigi** [comment]
Fixed on 7.5.2.2. Please be aware that for Custom Themes you'll have to use the setting designated for this effect: 
https://material-theme.com/docs/configuration/custom-themes/#color-scheme

**Comment by spencerhakim** [comment]
@mallowigi This is happening again on the latest version

**Comment by mallowigi** [comment]
> @mallowigi This is happening again on the latest version

Did you do what I wrote in the previous comment?

**Comment by spencerhakim** [comment]
I'm not using a custom theme, I'm on the free version. Am I to understand that choosing a non-Material color scheme is meant to be a premium feature?

**Comment by mallowigi** [comment]
No, it's not the problem here. So you're saying that changing the color scheme only is reset after restart, right?

**Comment by mallowigi** [comment]
I've just checked and it looks fine from my point of view

https://user-images.githubusercontent.com/5015756/191333069-40279dce-81bd-4d3b-b4bd-4f66e6627d75.mp4

or isn't it the scenario you were talking about?

**Comment by spencerhakim** [comment]
I was reproducing it earlier by closing/reopening the project I was working on but NOT quitting out of IntelliJ completely, and that was triggering the behavior. However, after completely quitting IntelliJ and opening it fresh, I can no longer reproduce the behavior. Might have been some weird edge case that only triggers on the first startup after upgrading? Either way, doesn't seem to be an issue any longer, sorry for bothering you.
